### DISCUSSION TITLE
Tasmota ESP32C3

### DISCUSSION THREAD

**Discussion by ghost** [arendst/Tasmota#20987] [Support request]
My ESP32C3 runs Tasmota 13.2.0 and "script". I read my SmartMeter and also use GoogleChart. Then suddenly there is a restart and "script" is not activated. The "script" runs for about 16-20 hours and then the restart. The variables will not be updated until the script is activated again. How can I prevent the reboot or enable "script" automatically after a reboot?

**Comment by sfromis** [comment]
Don't start another thread for the same question. https://github.com/arendst/Tasmota/discussions/20986
